src/components/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+class Button extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state= {name}.toString();
+    };
+}
```

Comment:
- [ ] The constructor should be receiving the props like this
```js
constructor(props) {
    super();
    this.props = props;
  }
```
- [ ] This component should return a `button` tag. e.g
```html
render() {
  return (
   <button type='button' name={this.props.name} className='btnColor'>{this.props.name}</button>
  );
}
````

Kindly modify